src/wallet/rpcwallet.cpp

```diff
@@ -2008,6 +2016,9 @@ Value encryptwallet(const Array& params, bool fHelp)
 
     if (fHelp)
         return true;
+    if (!fEnableWalletEncryption) {
+        return false;
```

Comment:
Should this throw a ``runtime_error``? I'm not sure what the observable behaviour from returning false is.